server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/GenerateAssertions.java

```diff
@@ -150,47 +177,66 @@ public String endpointBody(final Request the_request, final Response the_respons
     return my_endpoint_result.get();
   }
 
-  private Assertion assertionJSONToJava(AssertionResult assertionResult, List<String> candidates, String contestName, Long universeSize) {
-    String winner = candidates.get(assertionResult.getAssertion().getWinner());
-    String loser = candidates.get(assertionResult.getAssertion().getLoser());
-    Integer[] continuing = assertionResult.getAssertion().getContinuing();
-    Integer margin = assertionResult.getMargin();
-    double difficulty = assertionResult.getDifficulty();
 
-    Assertion assertion;
+  /* Gets all the choice names (i.e. candidate names) for a given contest from the database.
+   *
```

Comment:
For this and following methods, add more detail to comment (and use param/returns annotations).